paasta_tools/cli/cmds/status.py

```diff
@@ -158,7 +158,8 @@ def report_status_for_cluster(service, cluster, deploy_pipeline, actual_deployme
 
     if len(deployed_instances) > 0:
         status = execute_paasta_serviceinit_on_remote_master('status', cluster, service, ','.join(deployed_instances),
-                                                             system_paasta_config, stream=True, verbose=verbose)
+                                                             system_paasta_config, stream=True, verbose=verbose,
+                                                             ignore_ssh_output=True)
```

Comment:
It's a kwarg that gets used here: https://github.com/Yelp/paasta/pull/569/files/72466fe2a29c8a20deade5663dea95d2478812ea#diff-7d4e89c64360b719f36a1ca16d58607cR530